spec/datetimeformat.html

```diff
@@ -371,6 +371,11 @@ <h1>FormatDateTimePattern ( _dateTimeFormat_, _patternParts_, _x_, _rangeFormatO
             1. Let _f_ be the value of _dateTimeFormat_'s internal slot whose name is the Internal Slot column of the matching row.
             1. Let _fv_ be a String value representing the day period of _tm_ in the form given by _f_; the String value depends upon the implementation and the effective locale of _dateTimeFormat_.
             1. Append a new Record { [[Type]]: _p_, [[Value]]: _fv_ } as the last element of the list _result_.
+          1. Else if _p_ is equal to *"timeZoneName"*, then
```

Comment:
The super-long line starting `Else if _f_ is *"narrow"*,` below has an `If _p_ is *"timeZoneName"*,` sentence that's dead code now; please remove it.